Python-packages/covidcast-py/covidcast/plotting.py

```diff
@@ -170,6 +172,29 @@ def get_geo_df(data: pd.DataFrame,
     return output
 
 
+def animate(data: pd.DataFrame, filepath: str, fps: int = 3, dpi: int = 150, **kwargs) -> None:
+    """Generate an animated mp4 file of a signal over time.
+
+    :param data: DataFrame for a single signal over time.
+    :param filepath: Path where video will be saved.
+    :param fps: Frame rate in frames per second for animation. Defaults to 3.
+    :param dpi: Dots per inch for output video. Defaults to 150 on a 12.8x9.6 figure (1920x1440).
+    :return: None
+    """
+    # probesize is set to avoid warning by ffmpeg on frame rate up to 4k resolution.
+    writer = imageio.get_writer(filepath, fps=fps, input_params=["-probesize", "75M"])
+    num_days = (max(data.time_value) - min(data.time_value)).days
+    day_list = [min(data.time_value) + timedelta(days=x) for x in range(num_days+1)]
+    with tempfile.TemporaryDirectory() as tmpdirname:
+        for d in day_list:
+            print(f"Processing {d.date()}")
```

Comment:
If you don't mind another import, [tqdm](https://tqdm.github.io/) is a very nice progress bar for Python

Reply:
added